config/toml.go

```diff
@@ -508,7 +508,7 @@ unsafe-vote-timeout-delta-override = "{{ .Consensus.UnsafeVoteTimeoutDeltaOverri
 # This field configures how long the consensus engine will wait after receiving
 # +2/3 precommits before beginning the next height.
 # If this field is set to a value greater than 0, it will take effect.
-unsafe-commit-timeout-override = "{{ .Consensus.UnsafeCommitTimeoutOverride }}"
+# unsafe-commit-timeout-override = "{{ .Consensus.UnsafeCommitTimeoutOverride }}"
```

Comment:
We stopped setting this value so need to comment it out or else it'll cause a parsing error

Reply:
https://github.com/sei-protocol/sei-tendermint/commit/524337053db8771301bbc9cec54459e1fd937540